phpstan.neon

```diff
@@ -4,14 +4,15 @@ includes:
     - vendor/phpstan/phpstan-mockery/extension.neon
     - vendor/phpstan/phpstan-phpunit/extension.neon
     - phpstan-extension.neon
+    # TODO: Rewrite all phpstan rules in such a way that the node connecting visitor is no longer required
+    # TODO: Currently, \BrandEmbassyCodingStandard\PhpStan\Rules\Method\PhpUnitTestMethodRule retrieves
+    # TODO: attribute 'parent', which is provided by the node connecting visitor
+    - ./phpstan-node-connecting-visitor.neon
 
 parameters:
     level: max
 
-    featureToggles:
-        finalByPhpDocTag: true
```

Comment:
It seems that the featureToggle is not necessary anymore. Phpstan seem to check PhpDoc by default:

https://github.com/phpstan/phpstan-src/blob/8b99661ac959054c3b82bbf100e5158958245373/src/Rules/Classes/ExistingClassInClassExtendsRule.php#L77-L83

https://github.com/phpstan/phpstan-src/blob/8b99661ac959054c3b82bbf100e5158958245373/src/Reflection/ClassReflection.php#L1041-L1053

For context, this feature flag was added in #60 as we cannot safely use the `final` keyword. It would be nice to test that PhpStan will report an error when we try to extend a class with `@final` annotation.

Reply:
Good idea, I will test it when we will be clearing some sub-directories of phpstan issues on the platform-backend. :+1:

Reply:
@PetrHeinz I have verified that the @final annotation now is taken into account by phpstan without any feature toggle:
![image](https://user-images.githubusercontent.com/4999006/191909037-7aeaa5fe-f625-4646-9e18-d9e5dbf01785.png)